code/modules/mob/living/carbon/human/species.dm

```diff
@@ -977,11 +977,14 @@
 		radiation = 0
 		return TRUE
 
-	if(radiation > RAD_MOB_KNOCKDOWN)
+	if(radiation > RAD_MOB_KNOCKDOWN && prob(RAD_MOB_KNOCKDOWN_PROB))
 		if(!H.IsKnockdown())
 			H.emote("collapse")
 		H.Knockdown(RAD_KNOCKDOWN_TIME)
```

Comment:
I think you meant to change this to RAD_MOB_KNOCKDOWN_AMOUNT like you did for monkeys. In fact you could probably just change the value of RAD_KNOCKDOWN_TIME instead of introducing a new define.

Reply:
Oh, yeah I missed this somehow. Though I thought I remember changing this 😕